tools/clang/test/CodeGenHLSL/unroll/count_cbuff_br.hlsl

```diff
@@ -0,0 +1,26 @@
+// RUN: %dxc -E main -T ps_6_0 %s | FileCheck %s
+
+// Identical to count_cbuff.hlsl, except checks for number of br's
+
+// entry
+// CHECK: br
+// loop iteration
```

Comment:
You could make this a bit stronger by checking for calls to dot where you have the "loop iteration"